### ISSUE TITLE
MAL API it's broken: HTTP 404

### ISSUE THREAD

**Issue by ryukinix** [ryukinix/mal#100]
# Description
It's eternally dead? Will be shutdown forever?

We need research for info. I didn't find nothing yet.

Reading info in the subreddit of MyAnimeList seems that all 3rd apps are now broken due to the aggressive move of myanimelist shutdown it's own API. We need look for more info. 

# References
* https://www.reddit.com/r/MyAnimeList/comments/8lwhyt/so_poxkwt_mal_no_longer_works/
* https://www.reddit.com/r/MyAnimeList/comments/8lspub/problems_with_pocket_mal_and_other_3rd_party_apps/
* https://myanimelist.net/modules.php?go=api
* https://myanimelist.net/api

**Comment by kutsan** [comment]
Oh my. Literally the worst API in the universe. How can you shutdown without saying any word? No announcement, no nothing.

**Comment by ryukinix** [comment]
Some people are saying that this can be related to the new development of the official mobile application for MyAnimeList. This is a quite sad.

In other words, there is a new API running, because I doubt that they will create a mobile app officially using web scrapping. But even if they are building a new API, this doesn't means that they will make as public.

This is horrible. We should deserves at least a announcement, a warning for the API being dropped etc. Digging in the MAL forums someone in the staff said that the API will be disabled **INDEFINITELY**. What means: probably forever.

This is a shame.

**Comment by kutsan** [comment]
I completely agree with you. I sent a ticket to their support system about what's going on, if I get any useful response I'll post it here.

By the way, do you know any better service other than MyAnimeList that I can track of my anime/manga adventure? I don't want to support them and their horrible service anymore. I mean local sqlite database fits for this job though, I may try that.

**Comment by ryukinix** [comment]
> By the way, do you know any better service other than MyAnimeList that I can track of my anime/manga adventure? I don't want to support them and their horrible service anymore. I mean local sqlite database fits for this job though, I may try that.

I'm looking for a better solution as well, some people that I know talk to me about the https://anilist.co/ (which seems nice, but the dev seems a lot agressive and change the url and the internal system a LOT) and kitsu.io, which have a funny name.

I particurlly don't use any of them, but kitsu.io supports import all our list data from mal, which will save our time. I don't know yet if anilist has the same feature, though I liked more the anilist interface than kitsu.

**Comment by kutsan** [comment]
Cool, I didn't know about Kitsu. I will try them when I have time. Thanks!

**Comment by SolarAquarion** [comment]
Use Jikan? https://github.com/AWConant/jikanpy

**Comment by lubien** [comment]
Kitsu has the same problem as anilist. Dev keeps changing stuff. Used to be hummingbird and now thinga changed.

Not saying it's bad but either settle down with the my"nothing changes"animelist or the anilist"maybe changes"kitsu

Remember, you can import MAL list to anilist and kitsu but not the other way around.

There's notify.moe but things are in eternal refactoring beta.

**Comment by ryukinix** [comment]
An announcement from MAL staff. 

![announcement](https://i.imgur.com/rpwpk3q_d.jpg?maxwidth=640&shape=thumb&fidelity=medium)

**Comment by kutsan** [comment]
@ryukinix I can't view/access to the image you posted.

**Comment by lubien** [comment]
me too

**Comment by ryukinix** [comment]
Fixed

**Comment by ryukinix** [comment]
Credits to @lubien that found this post. Please, provide the source if you can.

**Comment by kutsan** [comment]
Found it https://myanimelist.net/forum/?topicid=1731860#msg55362465.

**Comment by lubien** [comment]
**Waits for 10 years**

MAL staff: just finishing some stuff before the new API closed beta

**Comment by kutsan** [comment]
> "I hope you will continue to put your trust in us and find our overall service rewarding and useful."

Ha ha, no thanks.

**Comment by ryukinix** [comment]
![image](https://user-images.githubusercontent.com/7642878/40667524-e8edb6f6-6351-11e8-9d8e-059f86c78b8c.png)


This is the second day.

**Comment by kutsan** [comment]
This is getting intense.

![image](https://user-images.githubusercontent.com/10108377/40718445-a9e3782a-6418-11e8-891b-79d105eebaf1.png)

This incident made me see there are how many MAL fanboys out there.

**Comment by lubien** [comment]
everyday we stray further from god

**Comment by kutsan** [comment]
This is my favorite anime now. Can't wait to see the 3rd episode.

**Comment by lubien** [comment]
Konno Mai Anime Risuto Ga Konna Ni Mentenansu Wake Ga Nai

**Comment by ryukinix** [comment]
@lubien weeb

**Comment by kutsan** [comment]
Just for your information, I started using Kitsu and apparently [their source code](https://github.com/hummingbird-me) is completely open-source, both server and client side. I also found [trackma](https://github.com/z411/trackma) CLI for an alternative to this repository, it even has curses interface, Kitsu integration, auto-increase episodes count after finishing an episode and such features.

**Comment by ryukinix** [comment]
Trackman seems pretty neat, thanks for sharing @kutsan

**Comment by ryukinix** [comment]
BTW, something from the official news:

![image](https://user-images.githubusercontent.com/7642878/40855661-a74053b0-65c4-11e8-9da1-8f97df8083fc.png)

**Comment by lubien** [comment]
![fb_img_1509028409311](https://user-images.githubusercontent.com/9121359/40861704-67e1a2ea-65c0-11e8-80f7-06835a968dbd.jpg)

**Comment by kutsan** [comment]
> Trackman seems pretty neat, thanks for sharing @kutsan

@ryukinix Yeah, I like it a lot too. You are welcome!

**Comment by lubien** [comment]
> ![image](https://user-images.githubusercontent.com/9121359/40869435-4b54081c-65f1-11e8-8546-80b1e34ccdd2.png)


![image](https://user-images.githubusercontent.com/9121359/40869436-4edd0006-65f1-11e8-89fa-b712c565f6c0.png)

**Comment by kutsan** [comment]
Now we can export our lists: https://myanimelist.net/announcement/201806

**Comment by bradenbest** [comment]
@kutsan Well, they lied.

https://myanimelist.net/animelist/b1kmusic
> This page is currently unavailable. Please click here for details.

**Comment by ryukinix** [comment]
It seems MAL new API it's more or less useful right now, but I'm not sure if I should resurrect that software with mal. Currently, I don't even use mal anymore, I use much more anilist.

If we resurrect that,  it would be nice if we can choose a mal data source provider between myanimelist and anilist. These days I've been using another software that works like this, but is implemented in Golang: https://github.com/aqatl/mal
